src/site/_data/contributors.js

```diff
@@ -203,6 +203,14 @@ const contributors = {
     twitter: 'bibydigital',
     github: 'mihajlija',
   },
+  mustafakurtuldu: {
```

Comment:
@mustafa-x do you have a personal site that you'd like to add to the list of social media links?

Reply:
Added https://www.designtoday.info/.

Reply:
that could work :)